ipaplatform/base/tasks.py

```diff
@@ -136,14 +136,9 @@ def modify_nsswitch_pam_stack(self, sssd, mkhomedir, statestore):
         If sssd flag is true, configure pam and nsswtich so that SSSD is used
         for retrieving user information and authentication.
 
-        Otherwise, configure pam and nsswitch to leverage pure LDAP.
-        """
-
-        raise NotImplementedError()
-
-    def modify_pam_to_use_krb5(self, statestore):
-        """
-        Configure pam stack to allow kerberos authentication.
+        Otherwise, this method might configure pam and nsswitch to leverage
+        pure LDAP and configure pam stack to allow kerberos authentication.
+        no-sssd option is deprication and will be removed soon.
```

Comment:
1. The comment is too vague to be useful (might configure? which fact would trigger configuration?) I'd rather write:
`Otherwise, if authconfig is used, configure pam and nsswitch to leverage pure LDAP.`
2. The part about kerberos authentication is not relevant to this method. It was done in modify_pam_to_use_krb5() which you removed.

Reply:
since this is a doc string in a base class, I think it is good to keep it as it is.  Current comment describes what method in derived class should implement.